### PR TITLE
Fix import case sensitivity for Unix

### PR DESCRIPTION
**Pull request by josephluck**

### CODE REVIEW

File: src/Mappings.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as patterns from './patterns';
+import * as patterns from './Patterns';
 import * as models   from './models';
```
Comment (ghost) [inline src/Mappings.js:2]: Looks like `models` should be PascalCase too

**Review by ghost — CHANGES_REQUESTED** [review]
Thanks joseph - looks like I changed the dir names to pascal at some point but didn't update some of the import paths. If you could double check that there aren't other examples of this, then happy to merge! Hope things are going well at Fantastec - heard great things from Rich. Cheers!

File: src/Mappings.js (hunk 1 above)
Comment (josephluck) [reply src/Mappings.js:2]: `models` is a lowercase directory with a default `index` import so no need to pascal case - however I can update the PR to PascalCase the directory names too if you prefer :+1:

**Comment by josephluck** [comment]
Thanks @patrickkunkadazn !

Just double checked and I can't see any other occurrences, should be gtg :tada: 

Things are going well :smile: thanks.

File: src/Mappings.js (hunk 1 above)
Comment (patrickkunka) [reply src/Mappings.js:2]: Oh yeah - don't know what I'm talking about, all good!

**Comment by patrickkunka** [comment]
Thanks @josephluck

**Pull request merged**